nautobot/ipam/migrations/0051_added_optional_vrf_relationship_to_vdc.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.19 on 2025-02-24 21:20
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("virtualization", "0030_alter_virtualmachine_local_config_context_data_owner_content_type_and_more"),
+        ("dcim", "0067_controllermanageddevicegroup_tenant"),
+        ("ipam", "0050_vlangroup_range"),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name="vrfdeviceassignment",
+            unique_together={("vrf", "device"), ("vrf", "virtual_machine")},
+        ),
```

Comment:
Why this?

Reply:
Just did a `nautobot-server makemigrations` here so it is autogenerated. I will delete it and see how it goes.